app/src/main/java/org/simple/clinic/crash/SentryCrashReporter.kt

```diff
@@ -30,7 +31,11 @@ class SentryCrashReporter @Inject constructor(
 ) : CrashReporter {
 
   override fun init(appContext: Application) {
-    Sentry.init(AndroidSentryClientFactory(appContext))
+    Sentry.init { options ->
+      options.dsn = SENTRY_DSN
+      options.sampleRate = SENTRY_SAMPLE_RATE
+      options.environment = SENTRY_ENVIRONMENT
```

Comment:
The `sentry.properties` file also has an auth token which should be a secret. Where is that getting passed?

Reply:
Do we really need it? The API doesn't expose a way to set that. As far as I can tell from the docs, I don't find it as needed for the Android implementation?

Reply:
Found the use case for it Gradle plugin. Adding it there
https://docs.sentry.io/platforms/android/proguard/